string action_name
int32 num_of_calls
float32 mean_duration
float32 max_duration